### PR TITLE
Collector release notes

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/installed-collectors/collector-installation-reference/advanced-ui-installer-settings.md [hunk 1]
```diff
@@ -16,6 +16,39 @@ These options appear if you click **Advanced Settings** on the Confirmation dial
...
 | Run As | Root/Administration User<br/>Custom User | User account under which the Collector runs on this machine.<br/>For Windows, the user account must have "Log on as Service" privileges. Default is the Administrator/Root user. |
 
+### Provide Full Control access for Custom User
+
+Follow the below steps to provide **Full Control** privileges for the Custom User (Non Admin/System user) with existing collector.
+
+1. Open the registry editor from the start menu.
+1. Navigate to the below path and search for **sumo-collector** folder.
+    `HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\sumo-collector`
+1. Right-click on the **sumo-collector** folder, select **Permissions** from the dropdown menu.<br/><img src={useBaseUrl('img/send-data/registry-editor.png')} alt="registry-editor" width="600" style={{border: '1px solid black'}} />
```
Comment (jpipkin1) [inline docs/send-data/installed-collectors/collector-installation-reference/advanced-ui-installer-settings.md:26]:
```suggestion
1. Right-click on the **sumo-collector** folder and select **Permissions** from the dropdown menu.<br/><img src={useBaseUrl('img/send-data/registry-editor.png')} alt="registry-editor" width="600" style={{border: '1px solid black'}} />
```

File: docs/send-data/installed-collectors/collector-installation-reference/advanced-ui-installer-settings.md [hunk 2]
```diff
@@ -16,6 +16,39 @@ These options appear if you click **Advanced Settings** on the Confirmation dial
...
+1. In the **Permissions for sumo-collector** pop-up,
```
Comment (jpipkin1) [inline docs/send-data/installed-collectors/collector-installation-reference/advanced-ui-installer-settings.md:27]:
```suggestion
1. In the **Permissions for sumo-collector** pop-up:
```

File: docs/send-data/installed-collectors/collector-installation-reference/advanced-ui-installer-settings.md [hunk 3]
```diff
@@ -16,6 +16,39 @@ These options appear if you click **Advanced Settings** on the Confirmation dial
...
+    1. Select the **Full Control** **Allow** checkbox.
+    1. Click on **Apply**.
+    <br/><img src={useBaseUrl('img/send-data/permissions.png')} alt="permissions" style={{border: '1px solid black'}} width="300" />
+
+Follow the below steps to provide **Full Control** privileges for the Custom User (Non Admin/System user) with newly installed collector.
+
+1. After the collector is installed, open the registry editor from the start menu.
+1. Navigate to the below path and search for **sumo-collector** folder.
+    `HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\sumo-collector`
+1. Right-click on the **sumo-collector** folder, select **Permissions** from the dropdown menu.<br/><img src={useBaseUrl('img/send-data/registry-editor.png')} alt="registry-editor" width="600" style={{border: '1px solid black'}} />
```
Comment (jpipkin1) [inline docs/send-data/installed-collectors/collector-installation-reference/advanced-ui-installer-settings.md:38]:
```suggestion
1. Right-click on the **sumo-collector** folder and select **Permissions** from the dropdown menu.<br/><img src={useBaseUrl('img/send-data/registry-editor.png')} alt="registry-editor" width="600" style={{border: '1px solid black'}} />
```

File: docs/send-data/installed-collectors/collector-installation-reference/advanced-ui-installer-settings.md [hunk 4]
```diff
@@ -16,6 +16,39 @@ These options appear if you click **Advanced Settings** on the Confirmation dial
...
+1. In the **Permissions for sumo-collector** pop-up,
```
Comment (jpipkin1) [inline docs/send-data/installed-collectors/collector-installation-reference/advanced-ui-installer-settings.md:39]:
```suggestion
1. In the **Permissions for sumo-collector** pop-up:
```

**Review by jpipkin1 — APPROVED** [review]
Looks good, @JV0812. I have minor suggestions.

**Comment by JV0812** [comment]
Thanks John! I have fixed your comments.

**Pull request merged**